### PR TITLE
Loading indicator custom element.

### PR DESCRIPTION
**Pull request by greenjeffreym**

### CODE REVIEW

File: src/components/loaders/loaders.jsx [hunk 1]
```diff
@@ -6,7 +6,10 @@ module.exports = React.createClass({
 			<div>
 				<div className="cps-flexible-focus cps-card">
 					<div className="cps-card__header cps-subheader">
-						Loading indicators
+						Loading indicators: 
+						<p className="cps-body">
+						These will work although they are depricated now in favor of the custom element <a href="https://github.com/CanopyTax/canopy-styleguide/blob/master/src/custom-elements/cps-loader"> CprLoader</a>
```
Comment (kentmclean) [inline src/components/loaders/loaders.jsx]: small thing: typo on `depricated` should be `deprecated`

File: src/custom-elements/cps-loader/Readme.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# cps-loader
+Provieds a loading indicator with 3 dots
```
Comment (kentmclean) [inline src/custom-elements/cps-loader/Readme.md]: typo on `Provieds`; should be `Provides`

File: src/custom-elements/cps-loader/Readme.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+## HTML Usage
+```html
+<CprLoader dotSize="24" page="true" color="orange"/>
```
Comment (kentmclean) [inline src/custom-elements/cps-loader/Readme.md]: You may want to also include the non-React usage here

File: src/components/loaders/loaders.jsx (hunk 1 above)
Comment (greenjeffreym) [inline src/components/loaders/loaders.jsx]: done

File: src/custom-elements/cps-loader/Readme.md (hunk 2 above)
Comment (greenjeffreym) [inline src/custom-elements/cps-loader/Readme.md]: done

File: src/custom-elements/cps-loader/Readme.md (hunk 3 above)
Comment (greenjeffreym) [inline src/custom-elements/cps-loader/Readme.md]: done

**Pull request merged**